app/models/organisation.rb

```diff
@@ -4,6 +4,9 @@ class Organisation < ApplicationRecord
 
   validates :rdv_solidarites_organisation_id, uniqueness: true, allow_nil: true
   validates :name, presence: true
+  validates :phone_number, length: { is: 10 },
+                           format: { with: /\d+/, message: "Le numéro ne doit contenir que 10 chiffres" },
+                           allow_nil: true
```

Comment:
du coup no need for this, il faudrait plutôt le valider le format du numéro (`HasPhoneNumberConcern`), mais c'est pas le scope de cette PR.